src/runner/browser-env/vite/plugins/generate-index-html.ts

```diff
@@ -129,6 +129,16 @@ export const plugin = async (): Promise<Plugin[]> => {
                     return ["export default () => {};", "export const resolve = () => ''"].join("\n");
                 }
             },
+
+            transform(code, id): Rollup.TransformResult {
+                if (id.includes(".vite/deps/expect.js")) {
+                    return {
+                        code: code.replace("var fs = _interopRequireWildcard(require_graceful_fs());", "var fs = {};"),
```

Comment:
Here is hack in order to correctly use `expect` in browser environment. I'm replacing graceful fs here